content/en/docs/contribute/participate/roles-and-responsibilties.md

```diff
@@ -188,4 +188,10 @@ To apply:
 
 2. Assign the PR to one or more current SIG Docs approvers.
 
-If approved, a SIG Docs lead adds you to the appropriate GitHub team. Once added, [K8s-ci-robot](https://github.com/kubernetes/test-infra/tree/master/prow#bots-home) assigns and suggests you as a reviewer on new pull requests.
\ No newline at end of file
+If approved, a SIG Docs lead adds you to the appropriate GitHub team. Once added, [K8s-ci-robot](https://github.com/kubernetes/test-infra/tree/master/prow#bots-home) assigns and suggests you as a reviewer on new pull requests.
```

Comment:
```suggestion
If approved, a SIG Docs lead adds you to the appropriate GitHub team. Once added, [@k8s-ci-robot](https://github.com/kubernetes/test-infra/tree/master/prow#bots-home) assigns and suggests you as a reviewer on new pull requests.
```
Could also link to https://github.com/k8s-ci-robot (which then links to https://github.com/kubernetes/test-infra/tree/master/prow#bots-home)